traitsui/qt4/video_editor.py

```diff
@@ -181,6 +181,10 @@ class VideoEditor(Editor):
     #: Synchronized to the trait named by factory.image_func.
     image_func = Callable()
 
+    #: The change in position required for an update to be emitted.
+    #: Synchronized to the trait named by factory.notify_interval.
```

Comment:
Should document the time units.

If the units aren't seconds, then it's probably worth having this value in seconds and then converting to Qt's preferred units (I _think_ you're doing that).

Reply:
This is documented in the factory